rusk-abi/Cargo.toml

```diff
@@ -17,8 +17,8 @@ dusk-bytes = "0.1"
 bytecheck = { version = "0.6", default-features = false }
 dusk-plonk = { version = "0.19", default-features = false, features = ["rkyv-impl", "alloc"] }
 
-piecrust-uplink = { version = "0.12" }
-piecrust = { version = "0.19", optional = true }
```

Comment:
should the rusk-abi CHANGELOG reflect these changes?

Reply:
Yes, you're right. Updated